x/decentralized-vault/keeper/grpc_query.go

```diff
@@ -0,0 +1,73 @@
+package keeper
+
+import (
+	"context"
+
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/status"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	ununifitypes "github.com/UnUniFi/chain/types"
+	"github.com/UnUniFi/chain/x/decentralized-vault/types"
+)
+
+var _ types.QueryServer = Keeper{}
+
+func (k Keeper) TransferRequestedNfts(c context.Context, req *types.QueryTransferRequestedNftsRequest) (*types.QueryTransferRequestedNftsResponse, error) {
+	var limit int
+	if int(req.NftLimit) > 0 {
+		limit = int(req.NftLimit)
+	} else {
+		limit = 10
+	}
+	ctx := sdk.UnwrapSDKContext(c)
+	requsts, err := k.GetTransferRequests(ctx, limit)
+	if err != nil {
+		return nil, err
+	}
+
+	var transferRequestNfts []*types.QueryTransferRequestedNftResponse
+	for _, v := range requsts {
+		transferRequestNft, err := k.makeTransferRequestedNftResponse(ctx, v)
+		if err != nil {
+			return nil, status.Error(codes.NotFound, "not found nft")
+		}
+		transferRequestNfts = append(transferRequestNfts, transferRequestNft)
+	}
+
+	return &types.QueryTransferRequestedNftsResponse{
+		Nfts: transferRequestNfts,
+	}, nil
+}
+
+func (k Keeper) TransferRequestedNft(c context.Context, req *types.QueryTransferRequestedNftRequest) (*types.QueryTransferRequestedNftResponse, error) {
+	if req == nil {
+		return nil, status.Error(codes.InvalidArgument, "invalid request")
+	}
+
+	ctx := sdk.UnwrapSDKContext(c)
+	transferReq, err := k.GetTransferRequestByIdBytes(ctx, []byte(req.NftId))
+	if (transferReq == types.TransferRequest{} || err != nil) {
```

Comment:
Can you remove `()`?

Reply:
I can't.
Because if I remove it, I will get a syntax error.

Reply:
Hmm - not a big problem